src/pages/articles/machine-learning/supervised-learning/index.md

```diff
@@ -1,14 +1,15 @@
 ---
 title: Supervised Learning
 ---
-#### Suggested Reading:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+In supervised learning, we know what the correct output should be. Supervised learning problems can be categorized into regression and classification. A regression problem is where you map input to a continuous output. A classification problem, in contrast, is where you map (group) inputs into discrete categories.
 
-- https://en.wikipedia.org/wiki/Supervised_learning
-- https://stackoverflow.com/a/1854449/6873133
+### Regression
+
+Given data about the mileage of used car, you can predict their prices. Since price is a continuous variable, this is a regression problem. In another example, Microsoft released a web app that predicts age based on picture. Again, as age is continuous rather than discrete or categorical, this is a regression problem.  
```

Comment:
I like the example, but when it says, "...you can predict their prices", I think it is a little bit unclear on which "prices" it is referring to. I know it is referring to the price of the car, but skimming quickly, it might refer to the price of gas. Just a suggestion 😄